airbyte-integrations/connectors/source-xero/source_xero/oauth.py

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import base64
+import logging
+from typing import Any, List, Mapping, MutableMapping, Tuple
+
+import pendulum
+import requests
+from airbyte_cdk.sources.streams.http.requests_native_auth import Oauth2Authenticator
+
+logger = logging.getLogger("airbyte")
+
+
+class XeroCustomCuonnectionsOauth2Authenticator(Oauth2Authenticator):
```

Comment:
You have a typo here - XeroCustomC**u**onnectionsOauth2Authenticator

Reply:
@alkali correct this and the connector is ready to go :rocket:

Reply:
done